WalletWasabi.Fluent/ViewModels/AddWallet/AddedWalletPageViewModel.cs

```diff
@@ -33,6 +33,8 @@ public AddedWalletPageViewModel(KeyManager keyManager)
 
 		private void OnNext()
 		{
+			Services.UiConfig.Oobe = false;
```

Comment:
yes, its needed to get rid of the welcome screen background during setup!

Reply:
It only actual has noticeable behavior if you have a clean install with no wallets.

Reply:
Sorry again... if you make these changes 
https://github.com/zkSNACKs/WalletWasabi/pull/5913#discussion_r699864040
https://github.com/zkSNACKs/WalletWasabi/pull/5913#discussion_r699864561

Then no need to set it here. because the dialog will set the oobe to false.

Reply:
removed.